src/SchemaView.tsx

```diff
@@ -100,11 +114,15 @@ export class SchemaViewWR extends React.PureComponent<SchemaViewProps> {
           schema={currentSchema}
           lookup={lookup}
           stage="both"
+          onSelectValidationRange={(range) => this.setState({ selectedValidationRange: range })}
+          validationResults={this.state.validationResults}
         />
         <SchemaViewWR.EditorContainer>
           <SchemaEditor
             initialContent={fullExample}
             schema={schema}
+            validationRange={this.state.selectedValidationRange}
```

Comment:
Sweet. I can see the state passthrough from the Editor to the Viewer. Interestingly, this is arguably a smell, the editor component should probably subsume the validation results as well inside of itself as I mentioned in another comment.